### PR TITLE
re-init two guides, closing other PR

### PR DESCRIPTION
**Pull request by dexhorthy**

### CODE REVIEW

File: initcontainer-logs/manifests/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+Troubleshoting example: Multi-container Deployments
```
Comment (dexhorthy) [inline initcontainer-logs/manifests/README.md]: needs changes from Prev PR

File: postgres-snapshots/manifests/pg-consumer.yaml [hunk 2]
```diff
@@ -31,29 +31,6 @@ spec:
...
-                  name: postgres-secret
-                  key: DB_PASSWORD
-            - name: DB_NAME
-              valueFrom:
-                secretKeyRef:
-                  name: postgres-secret
-                  key: DB_NAME
+          envFrom:
+            - secretRef:
+                name: postgres
```
Comment (dexhorthy) [inline postgres-snapshots/manifests/pg-consumer.yaml:36]: updated this to match the concise version from the guide

Comment (austinchambers) [reply postgres-snapshots/manifests/pg-consumer.yaml:36]: Actually, I thought about this and felt that it's good to have both versions. This one makes it more explicit what's in the secret, which is why I left it as is and used secretRef in the pg-snapshot pod. It's a minor thing though

Comment (dexhorthy) [reply postgres-snapshots/manifests/pg-consumer.yaml:36]: I think if you're referencing the values of a secret, you might as well also reference the keys, but I see your argument -- having it the longer way communicates the structure of the vars without having to switch files. Is it okay if I leave this as is for this example?

**Review by austinchambers — APPROVED** [review]

**Pull request merged**